core/src/main/java/org/eclipse/hono/util/Constants.java

```diff
@@ -216,6 +216,11 @@ public Authorities getAuthorities() {
      */
     public static final String HEADER_COMMAND = "hono-command";
 
+    /**
+     * The header name defined for setting the <em>one-way command</em> that is sent to the device.
+     */
+    public static final String HEADER_ONE_WAY_COMMAND = "hono-one-way-command";
```

Comment:
do we need this?

Reply:
Good point: this is only needed in the HTTP adapter implementation, so I will remove it from this PR (and provide it again in the next PR).

Reply:
What does the HTTP adapter need this for?

Reply:
If we skip the `command-request-id` for one-way commands, we do not need it (and send one-way commands only by omitting the `command-request-id`).
So, I will remove it completely.

Reply:
sounds good to me